src/main/java/homework_3/ImmutableClassTerminator.java

```diff
@@ -0,0 +1,55 @@
+package homework_3;
+
+/*                                  How to make Immutable Objects:
+    1. Don't provide "setter" methods — methods that modify fields or objects referred to by fields.
+    2. Make all fields final and private.
+    3. Don't allow subclasses to override methods. The simplest way to do this is to declare the class as final.
+       A more sophisticated approach is to make the constructor private and construct instances in factory methods.
+    4. If the instance fields include references to mutable objects, don't allow those objects to be changed:
+         - Don't provide methods that modify the mutable objects.
+         - Don't share references to the mutable objects. Never store references to external, mutable objects passed to
+           the constructor; if necessary, create copies, and store references to the copies. Similarly, create copies
+           of your internal mutable objects when necessary to avoid returning the originals in your methods.
+*/
+
+import java.util.Date;
+
+public final class ImmutableClassTerminator {
```

Comment:
Your class is mutable 
![image](https://user-images.githubusercontent.com/58872509/134056143-917c1b49-0b6f-46ab-bf2a-9e0bc6250eea.png)